options.go

```diff
@@ -208,6 +208,25 @@ func ConfigureHelp(options HelpOptions) Option {
 	})
 }
 
+// Group holds metadata about a node group used when printing help.
+type Group struct {
```

Comment:
This should go into `model.go`, and be returned by the existing group related model methods.

Reply:
Just to clarify, you want something like this?

```diff
type Node struct {
    ...
-    Group string  
+    Group *Group
}

type Group struct {
+   // Key is the `group` field tag value used to identify this group.
+   Key string
    Title string
    Header string
}
```

And have the key -> `Group` resolved during parsing in `build.go`?

Reply:
Exactly.